app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,105 @@
+class ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Runner <
+  ManageIQ::Providers::BaseManager::EventCatcher::Runner
+
+  TAG_TYPE = "type".freeze
+  TAG_SEVERITY = "severity".freeze
+  TAG_CLUSTER = "clusterid".freeze # collect only alerts with this tag
+
+  # keys are content of the tag 'type'
+  TARGETS = {
+    :node => {
+      :target_class => ContainerNode,
+      :target_key   => :name,
+      :source_key   => 'nodename'
+    }
+  }.freeze
+
+  def initialize(cfg = {})
+    super
+  end
+
+  def reset_event_monitor_handle
+    @event_monitor_handle = nil
+  end
+
+  def whitelist?(event)
+    tags = event.tags
+    tags && !tags[TAG_CLUSTER].nil?
+  end
+
+  def stop_event_monitor
+    @event_monitor_handle.try(:stop)
+  ensure
+    reset_event_monitor_handle
+  end
+
+  def monitor_events
+    event_monitor_handle.start
+    event_monitor_handle.each_batch do |events|
+      event_monitor_running
+      new_events = events.select { |e| whitelist?(e) }
```

Comment:
Following middleware collection I should now extract. They are using both black & white listing.
I guess we would just not define a black list for now.